packages/apps/tools/src/pages/transactions/cross-chain-transfer-tracker/index.tsx

```diff
@@ -229,12 +226,11 @@ const CrossChainTransferTracker: FC = () => {
             <Grid>
               <GridItem>
                 <RequestKeyField
-                  helperText={inputError || undefined}
-                  status={validRequestKey}
+                  errorMessage={inputError || errors.requestKey?.message}
+                  isInvalid={validRequestKey}
```

Comment:
Can we rename `validRequestKey` to `invalidRequestKey` or make it `isInvalid={!validRequestKey}`. Now it doesn't look logical

Reply:
those are really spesfic refactoring, I just had to make a minimal change to get the stuff working, we can refactor all of these later